.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -156,9 +156,15 @@ See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on
         </a>
       </td>
       <td>
-        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Dataflow/lastCompletedBuild/">
-          <img alt="Build Status" src="https://ci-beam.apache.org/job/beam_PostCommit_XVR_Dataflow/lastCompletedBuild/badge/icon">
-        </a>
+        <a href="https://ci-beam.apache.org/job/beam_PostCommit_XVR_PythonUsingJava_Dataflow/lastCompletedBuild/">
```

Comment:
To clarify, we are splitting Python SQL and not-SQL tests to different test suites here ?

Reply:
Yes. For other runners, a single XVR test covers all three test suites: PythonUsingJava, PythonUsingJavaSQL and JavaUsingPython. For Dataflow, each suite spends more time (because of Dataflow job launching delay) so those three test suites are separated.

Reply:
Thanks. Let's also add appropriate test triggers, if we don't already have them.